ProjectFiles/Assets/Scripts/Game/RoundManager.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using System.Collections.Generic;
+using System.Timers;
+using Network;
+
+namespace Game
+{
+    static class RoundTimings
+    {
+        // All times in seconds
+        public const ushort PreRoundLength = 10;
+        public const ushort RoundLength = 45;
+    }
+    
+    public class RoundManager
+    {
+        public bool Started { get; private set; }
+        private World world;
+        private ushort roundNumber = 0;
+        
+        // Timer to countdown to the start of the round.
+        private Timer roundTimer;
+        private ushort preRoundLength;
+        private ushort roundLength;
+
+        public event GameStartDelegate GameStartEvent;
+        public event PreRoundStartDelegate PreRoundStartEvent;
+        public event RoundStartDelegate RoundStartEvent;
+        public event RoundEndDelegate RoundEndEvent;
+        public event EliminatePlayersDelegate EliminatePlayersEvent;
+
+        public RoundManager(World world)
+        {
+            this.world = world;
+        }
+        
+        public void StartGame()
+        {
+            Started = true;
+            NotifyGameStart();
+            preRoundLength = RoundTimings.PreRoundLength;
+            roundLength = RoundTimings.RoundLength;
+
+            StartPreRound();
+        }
+
+        public void StartPreRound()
+        {
+            List<ushort> activeSpaces = new List<ushort>();
```

Comment:
Maybe a set, as a minor thing

Reply:
Yeah fairs, we can look into it in more detail when we get to developing the system that selects which spaces are active :smile: